### DISCUSSION TITLE
Alternative protocol: TMTP in Tutanota

### DISCUSSION THREAD

**Discussion by ghost** [tutao/tutanota#4289] [Ideas]
Hi all!

#### 1. feature-name
Alternative protocol: TMTP in Tutanota

#### 2. feature-description
1. I recently found this network protocol and I would like to know more if it is more secure or not compared to SMTP. 
2. I believe that well-known projects such as Protonmail, Tutanota etc... can help clarify my doubt as they are known as email providers. 
3. Particularly, I use Tutanota. Personally, I use Protonmail. 
4. I would like to know something... what do you think about the TMTP protocol as an alternative to the protocol: SMTP?

#### 3. Why? 
1. I believe that Protonmail and Tutanota are the best and most secure mail providers currently.
2. I would like to know what the community thinks of this new protocol for email as an alternative to SMTP
3. I'm doing academic research and I'm consulting the biggest open source email projects to find out if this protocol is safer or not compared to SMTP. If anyone could give any answers I would be happy - it would help the research I'm doing.
4. I'm a web developer, that is, I know things like HTML, CSS, JS and frameworks in js like react, vuejs etc. I have no knowledge in languages like go, the TMTP protocol was written in the language go.
5. I would like to help 2 open source communities: Tutanota and networkimprov.
6. I recently read the documentation on TMTP and found the proposal interesting. So... I including was opening an Issue asking if the networkimprov community is interested in creating an RFC on TMTP as an alternative to SMTP. Apparently this would be done more in the future and not now.
 
#### 4. Notes
1. Despite understanding the TMTP proposal, I still think it is necessary to hear other opinions about whether it is safe or not.
2. I'm not promoting or wanting to promote any product, company, person, technology, service, business. My purpose in referencing the links is for bibliography only.
3. This post is related to this main question I have and this idea

#### 5. Questions
1. If the TMTP protocol is more secure, would Tutanota use it?
2. What do you all think of the alternative TMTP protocol that replaces SMTP for email in Tutanota?

#### 6. References
- https://github.com/networkimprov/mnm/issues/10
- https://github.com/networkimprov/mnm/issues/12
- https://github.com/networkimprov/mnm/blob/master/Protocol.md
- https://mnmnotmail.org/
- https://github.com/networkimprov/mnm
- https://talk.commonmark.org/t/commonmark-in-emails/2942

**Comment by charlag** [comment]
Hi,
I can't speak for all and this is the first time I hear about TMTP. I personally had some curiosities about what can be done on top of ActivityPub or Matrix.

**Reply by ghost** [reply]
@charlag Hi! thank you for feedback ;D I'm happy with your comment

**Reply by ghost** [reply]
@charlag Hi! I found a request in Gnome for opening Geary with tmtp support: https://discourse.gnome.org/t/have-you-all-ever-heard-of-the-tmtp-protocol-as-an-alternative-to-smtp/10312 - Some comments are interesting because they talk about tmtp if it is a good protocol or not
